.github/workflows/ci.yml

```diff
@@ -4,14 +4,29 @@ on:
   pull_request:
 
 jobs:
+  get-branch-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get branch name
+        shell: bash
+        # The workflow is triggered by pull_request so we use `GITHUB_BASE_REF`
+        run: echo "branch_name=${GITHUB_BASE_REF}" >> $GITHUB_OUTPUT
+        id: get_branch_name
+    outputs:
+      branch_name: ${{ steps.get_branch_name.outputs.branch_name }}
+  
   test-cedar-policy:
     name: Test Rust implementation
-    uses: cedar-policy/cedar/.github/workflows/run_integration_tests_reusable.yml@main
+    needs: get-branch-name
+    uses: cedar-policy/cedar/.github/workflows/run_integration_tests_reusable.yml@release/3.2.x
     with:
       cedar_integration_tests_ref: ${{ github.ref }}
+      cedar_policy_ref: ${{ needs.get-branch-name.outputs.branch_name }}
```

Comment:
I'll be porting the `get-branch-name` change back to `main`.

The switch to the `release/3.2.x` version of the YAML file is required for versions 3.2.x and 3.3.x due to some weird behavior of cargo re deprecation warnings. The fix is the ` --force-warn deprecated` option introduced in https://github.com/cedar-policy/cedar/pull/852 (but only on the 3.2.x and 3.3.x branches).